.github/workflows/calibreapp-image-actions.yml

```diff
@@ -0,0 +1,12 @@
+name: Compress images
+on: pull_request
+jobs:
+  build:
+    name: calibreapp/image-actions
```

Comment:
Can you change the name to match as well? EG: Compress Images. This way it'll show up nicely in the UI =)